.github/workflows/remark.yml

```diff
@@ -0,0 +1,55 @@
+name: Remark
+
+on:
+  push:
+    branches:
+      - auto
+      - try
+  pull_request:
+    paths:
+    - '**.md'
```

Comment:
You also may want to exclude `gcc/config/**/*.md` so that all the machine description files won't get checked.